modules/stats.php

```diff
@@ -893,13 +893,6 @@ function stats_dashboard_head() { ?>
 /* <![CDATA[ */
 jQuery(window).load( function() {
 	jQuery( function($) {
```

Comment:
Hmm, question about this... why are we wrapping the `.load` callback code with `jQuery( function( $ ) { } );` ? I don't know if I'm missing something, but seems superfluous to me in this case